File: Algorithm/Backjun/bronze5/b10950/b10950.py
# A+B -3
# https://www.acmicpc.net/problem/10950

# 두 수를 더하는 함수
def getSum(a, b):
    return a + b

answers = []

n = int(input())
for i in range(n):
    a, b = map(int, input().split())
    answers.append(getSum(a, b))

for answer in answers:
    print(answer)

File: Algorithm/Backjun/bronze5/b10950/b10950.js
// A+B -3
// https://www.acmicpc.net/problem/10950

const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
})

let input = []

// 두 수를 더하는 함수
function getSum(a, b) {
  return a + b
}

rl.on('line', (line) => {
  input.push(line)
})

rl.on('close', () => {
  const n = parseInt(input[0])
  let answer = []

  for (let i = 1; i <= n; i++) {
    const [a, b] = input[i].split(' ').map(Number)
    answer.push(getSum(a, b))
  }

  answer.forEach(answer => console.log(answer))
})
